po/pt-BR.po

```diff
@@ -5750,25 +5784,28 @@ msgid ""
 "In the example, `pets` is allocated on the stack and the vector data is on "
 "the heap. The two vector elements are _fat pointers_:"
 msgstr ""
+"No exemplo, `pets` é alocado na pilha e os dados do vetor estão no heap. Os "
```

Comment:
should we translate terms like stack?

Reply:
We try to stick as close as possible to the Rust book pt-BR translation: https://rust-br.github.io/rust-book-pt-br/